api/leaderboard.go

```diff
@@ -442,18 +440,12 @@ func (app *App) GetAroundMember(ctx context.Context, req *api.GetAroundMemberReq
 	if tenantID != nil {
 		result, err := app.Enricher.Enrich(ctx, tenantID[0], req.LeaderboardId, members)
```

Comment:
Instead of having line 448, what about this?

```suggestion
		members, err := app.Enricher.Enrich(ctx, tenantID[0], req.LeaderboardId, members)
```